### DISCUSSION TITLE
Improvements in reports

### DISCUSSION THREAD

**Discussion by jihee-k** [chatwoot/chatwoot#9734] [Feature Requests]
### There is a report download error problem.

There is a problem in downloading data for each item, and there is no data that can be downloaded and verified smoothly. We have the following problems.

1. Download Agent Report → Information indistinguishable
2. Download CSAT Reports → Information indistinguishable
3. Download label reports → not downloaded (comprehensive data download should be possible, not by each label)
4. Download inbox reports → Information indistinguishable (I want it to be a comprehensive data download, not each inbox.)
5. Download team reports → Information indistinguishable
6. Download Message Reports tab → agent report down (We should be able to download the conversation.)
- is there a way to download the message data Data I talked to the customer ?

### Describe the solution you'd like

1. Every item's data should be downloaded,
2. You must be able to download the message data you exchanged with the customer.

**Comment by pranavrajs** [comment]
@jihee-k I'm not sure if I understand the issue. What do you mean by "Information indistinguishable"? Can you please provide screenshot or more information on what the expectation is?

**Comment by jihee-k** [comment]
@pranavrajs 
Please refer to the agent report. Only time appears for download for agent reports.
I would like to know more various information such as cast score, what label was processed, etc.
 
<img width="290" alt="image" src="https://github.com/chatwoot/chatwoot/assets/172241637/b8ae53fc-a6f6-420c-b7d0-c38d5c24788a">

**Comment by jihee-k** [comment]
@pranavrajs Can you check it again? I'll post the report again if I need any information

We need more information about the report download.

**Comment by pranavrajs** [comment]
@jihee-k Moved this to a discussion as there is immediate action item on this issue. To understand the request better, could you please share a sample excel file which explains you expectations.
